packages/vuepress-theme-nimatron/styles/_samples.scss

```diff
@@ -247,8 +237,9 @@ $spacing-units: (
 
 .sample-token {
   display: block;
-  height: 7px;
-  margin: $spacing-xs auto 0;
+  width: 1em;
```

Comment:
nit: could we/should we use a spacing unit here?

Reply:
`$spacing-m` looked too big, since the line-height is so much larger than the text is optically. :( 

This felt like something safe with good rendered results.